docs/ERROR_HANDLING.md

```diff
@@ -0,0 +1,70 @@
+# Stark Default Error Handling
+
+Stark provides its own error handler, StarkErrorHandler, that overrides [ Angular's default ErrorHandler](https://angular.io/api/core/ErrorHandler).
+ 
+This handler will dispatch a StarkUnhandledError action  to the application Store which you can treat as you want. 
+
+## StarkErrorHandlerModule
+
+If you want to use this handler, you simply have to import StarkErrorHandlerModule from stark-core in your `app.module.ts` file.
+
+
+``` typescript
+import {StarkErrorHandlerModule} from "@nationalbankbelgium/stark-core";
+
+@NgModule({
+	bootstrap: ...
+	declarations: ...
+	imports: [
+		StarkErrorHandlerModule.forRoot(),
+		...
+		]
+	})
+```
+
+## Effects definition
+
+To define what to do when an error happens, you can simply create a new effects file, like in the following example:
+
+```` typescript
+...
+import {StarkErrorHandlerActionTypes, StarkUnhandledError} from "@nationalbankbelgium/stark-core";
+
+/**
+ * This class is used to determine what to do with an error
+ */
+@Injectable()
+export class StarkErrorHandlerEffects {
+	
+	public constructor(
+		private actions$: Actions
+	) {}
+
+	public doSomething(): Observable<void> {
```

Comment:
The `@Effect()` should be kept in here as before... just the `{ dispatch: false }` should not be in there ;)